bakery-src/scripts/fetch_map_resources.py

```diff
@@ -14,21 +13,8 @@
 # relative links must work both locally, on PDF, and on REX, and images are
 # uploaded with the prefix 'resources/' in S3 for REX
 # so the output directory name MUST be resources
-RESOURCES_DIR_NAME = 'resources'
-
-
-def create_json_metadata(output_dir, sha1, mime_type, s3_md5, original_name, width, height):
-    """ Create json with MIME type and other metadata of resource file """
-    data = {}
-    data['original_name'] = original_name
-    data['mime_type'] = mime_type
-    data['s3_md5'] = s3_md5
-    data['sha1'] = sha1
-    data['width'] = width
-    data['height'] = height
-    json_file = output_dir / f'{sha1}.json'
-    with json_file.open(mode='w') as outfile:
-        json.dump(data, outfile)
+RESOURCES_DIR_NAME = 'initial-resources'
```

Comment:
Rather than hardcoding these directory names again, could they be passed in as commandline arguments (or require pulling them from the environment variables `IO_INITIAL_RESOURCES`?

Reply:
will change that